developers/index.markdown

```diff
@@ -18,54 +31,50 @@ To start working on \BlueLaTeX, you will need to execute these commands:
 ```shell
 $ git clone git@github.com:gnieh/bluelatex.git
 $ cd bluelatex
-$ sbt
-> compile
+$ sbt compile
 ```
 
 This will download all necessary dependencies, compile the entire project and should simply work.
 
-Sbt has a great feature for incremental compilation, you can simply launch the command prefixed with a `~`
+## Launching Test Server
 
-```shell
-> ~compile
+The sbt configuration of \BlueLaTeX allows you to start a test instance of the server. This test instance is launched as a daemon on Unix systems using [jsvc](http://commons.apache.org/proper/commons-daemon/jsvc.html) (this should be possible on windows too, but it is not tested yet, any help is welcome).
+
+First, create a file called `build.sbt` and add this line in it:
+
+```scala
+launchExe := "/usr/bin/jsvc"
 ```
+Or whatever path is returned by `which jsvc`
 
-And incremental compilation will be launched every time a file changes in the source tree.
+If you have your own running couchdb instance, we also recommend to add:
 
-## Launching Test Server
+You just need to add these lines in your build.sbt file:
 
-The sbt configuration of \BlueLaTeX allows you to start a test instance of the server. This test instance is launched as a daemon on Unix systems using [jsvc](http://commons.apache.org/proper/commons-daemon/jsvc.html) (this should be possible on windows too, but it is not tested yet, any help is welcome).
+```scala
+couchdb := None
 
-in the sbt console just type:
+couchPort := 5984
+```
+
+Then, to start the server, type:
```

Comment:
If this is to be the default setting, then it better be the default behavior, to avoid people having to override them after every clone.